Style/index.css

```diff
@@ -1,5 +1,52 @@
-#divBody{
-	padding-left: 20%;
-	padding-right: 20%;
-	padding-top: 2%;
+* {
+	margin: 0;
+	padding: 0;
+}
+
+h1 {
+	padding: 1em;
+}
+
+h1 img {
+	width: 100%;
+	margin: auto;
+	display: block;
+	max-width: 800px;
+}
+
+menu {
```

Comment:
I also renamed #menu to menu, cuz we can do that now: http://www.w3schools.com/tags/tag_menu.asp